tests/src/Api/TextIngestionApiTest.php

```diff
@@ -45,28 +45,20 @@ public function testTextIngestion(array $clientConfig, array $responses, $expect
                 ['group003', 'group004']
             );
         $this->assertEquals($expectedResult, $actualResult);
-    }
-
-    /**
-     * @param RequestInterface $request
-     */
-    public function inspectRequest(RequestInterface $request): void
-    {
-        if ($request->getUri() == 'http://example.com/token') {
-            $this->inspectTokenRequest($request);
-        } else {
-            $this->assertEquals('http://example.com/ingest/text?apiKey=bananas&database=cucumbers&uri=http%3A%2F%2Fexample.com&language=%5B%22en%22%2C%22ro%22%5D&reference=unique-my-ID', $request->getUri());
-            $this->inspectAuthorizationHeaders($request);
-            $this->inspectBoundary($request);
-            $parts = $this->getMultiParts($request);
-            $this->assertCount(6, $parts);
-            $this->inspectPart($parts[0], 'application/json', 'metadata', 55, '{"field1":["value1","value2"],"field2":["value3",2345]}');
-            $this->inspectPart($parts[1], 'application/json', 'aclUsers', 20, '["user001","user02"]');
-            $this->inspectPart($parts[2], 'application/json', 'aclNolUsers', 20, '["user003","user04"]');
-            $this->inspectPart($parts[3], 'application/json', 'aclGroups', 23, '["group001","group002"]');
-            $this->inspectPart($parts[4], 'application/json', 'aclNoGroups', 23, '["group003","group004"]');
-            $this->inspectPart($parts[5], 'text/plain', 'text', 43, 'The quick brown fox jumps over the lazy dog');
-        }
+        $request = $historyMiddleware->getHistoryContainer()[0]['request'];
```

Comment:
Always test also the request count, to make sure only the needed ones are performed.

Reply:
done